snow/engine/snowman/getter/getter.go

```diff
@@ -42,7 +42,7 @@ func New(
 
 	var err error
 	gh.getAncestorsBlks, err = metric.NewAverager(
-		"bs_get_ancestors_blks",
+		"get_ancestors_blks",
```

Comment:
This is renamed because we remove the duplicate `get_ancestors_blks` metric below